Fixing the hanging issue at Which Police Force

cmeney opened the pull request.

armoj commented:
I also don't see anything related to "email address" in the code changes. Do these changes enable a subsequent page view of the email address page or something?

cmeney commented:
There should be some additional content on the consent page that adds an email address just before the phone number. Is that not showing?

armoj commented:
Not that I can see, no. The only html page that you have altered in the PR is the `app/views/application/scotland-forces/index.html`. I cannot see any other page being altered - apart from the page you deleted.

Have you committed everything you needed to before you pushed?

jenthomson committed:
Fixed conflict issues

jenthomson committed:
Updates to content for the someone else minor concept

jenthomson committed:
Content updates for someone else / minor journey

jenthomson committed:
Updated the medical flow on the minors journey to fix bugs

jenthomson committed:
Fixed routing to only ask STI and Pregnancy question for VOSAAs

jenthomson committed:
Fixed the 'your' in skin and tissue injuries

murrlipp committed:
Content and concept updates

- Crime reference number flow in self-apply journey updated to match minors concept.
- Crime reference number content updates in main prototype self-apply and minors concept.
- Two placeholder concepts for equalities questions for discussion and updates later.

murrlipp committed:
Updated minor journey flow

Parents applying on behalf of minors updates (all content is draft):
- Content changes to divert non-parent applicants to OAS
- Content changes to clarify parental responsibility and shared parental rights
- Added context page to care/supervision order section

Added concepts:
- Occupation list (content TBC)
- Split money section into 'loss of earnings' and 'special expenses' (content TBC)

murrlipp committed:
Removed updates only relevant to minor concept

murrlipp committed:
Content updates

Updated content in parents applying on behalf of minors concept.

murrlipp committed:
Amended grammar error

murrlipp committed:
Added occupation list into split money journey concept

murrlipp committed:
Content updates

Content updated to suit minors

murrlipp committed:
Added WIP minors loss of earnings and special expenses section - content TBC

murrlipp committed:
Added draft content to app/views/concepts/minors-loe-se

murrlipp committed:
Added concepts

Minors loss of earnings and special expense options 1 and 2 added.

murrlipp committed:
Content updates to minors loss of earnings and special expenses section

murrlipp committed:
Content/flow updates

Content added, final three sections added to minor journey to create full MVP journey

murrlipp committed:
Minors impact concept added

Minor content updates in minors prototype

murrlipp committed:
Replaced "loss of earnings / special expenses" section with "impact" section

Updated content after review, added "sending documentation" paragraph to self-applicant journey to match minor journey.

murrlipp committed:
Routing amends

murrlipp committed:
Content updates to minor journey

murrlipp committed:
Impact sections contents updated, along with other content tweaks

murrlipp committed:
Content updates to minor journey

murrlipp committed:
Update index.html

jenthomson committed:
Updates to impact on child section in concepts

- removed third bullet on context page
- updated text on last line of text
- removed considering-loe-se page from flow (page still exists in folder)

jenthomson committed:
Fixed typo on impact context page in concepts

murrlipp committed:
Content/flow updates

murrlipp committed:
Merge branch 'master' of https://github.com/CriminalInjuriesCompensationAuthority/apply-for-compensation-prototype

murrlipp committed:
Minors LOE/SE concept and impact section updates

murrlipp committed:
Update index.html

murrlipp committed:
Minors injury section updated - infections and pregnancy

murrlipp committed:
Impact context screen content updated

murrlipp committed:
Routing and content updates

Added age routing and content to infections and pregnancy questions. Amended some routing in physical injuries section.

murrlipp committed:
Routing bugs fixed in injuries section

murrlipp committed:
Routing errors amended, char counts added to freetext boxes

murrlipp committed:
Flow and content updates

- Moved "does the child live at this address" to victim details section.
- Pregnancy context screens updated.

murrlipp committed:
Content tweaks, fixes to check your answers screen

murrlipp committed:
Content change to declaration screen

murrlipp committed:
Removed care order context screen, added declaration concepts

murrlipp committed:
Update error.html

murrlipp committed:
Amended routing on minors impact section concept option 3

murrlipp committed:
Declaration screen option updated

murrlipp committed:
Made CYA screens play back in order asked to user

murrlipp committed:
Amended adult journey impact section content - still TBC

murrlipp committed:
Content update to CYA screen

murrlipp committed:
Save and return concepts added

murrlipp committed:
Updated content, save and return concepts

murrlipp committed:
Save and return concepts updated

murrlipp committed:
Added task list functionality to self-applicant journey

Define a status for each section in the routing for entry and exit pages to that section, which updates the task list.  Task list screen can be found under app/views/application/task-list

Not yet integrated to journey.

murrlipp committed:
Merge pull request #274 from CriminalInjuriesCompensationAuthority/task-list

Added task list functionality to self-applicant journey

murrlipp committed:
Content updates

murrlipp committed:
Content updates, declaration routing updates

murrlipp committed:
Content update to fatal application question

murrlipp committed:
Fixed content on minor confirmation screen, updated content on adult confirmation screen

murrlipp committed:
Content updates

murrlipp committed:
Amended opening hours

murrlipp committed:
Routing and content changes

See change log for 23 Sep 2021.
https://apply-for-compensation-proto.herokuapp.com/#change-log

murrlipp committed:
Fixed typo

murrlipp committed:
Save and return MVP prototype added

murrlipp committed:
Updated minor journey address questions

See change log
https://apply-for-compensation-proto.herokuapp.com/#change-log

murrlipp committed:
Save and return prototype updates

murrlipp committed:
Updates to save and return prototypes

Secondary action now a link not a button due to accessibility.
System-generated password updated from three random words to 16 digit string.

murrlipp committed:
Error fix on save and return concept buttons

murrlipp committed:
Variable name updated in save and return prototype

murrlipp committed:
Added save and return prototype for testing

murrlipp committed:
Content updates

See changelog

murrlipp committed:
Fixed routing in minor journey

murrlipp committed:
Content updates to minor journey

murrlipp committed:
Content updates

See change log: https://apply-for-compensation-proto.herokuapp.com/#change-log

murrlipp committed:
Content updates

Content updates to minor journey based on legal and policy stakeholder feedback.

murrlipp committed:
Content updates to minor journey

Based on legal feedback

murrlipp committed:
Fixed routing on adult journey

GrahamBaird1 committed:
Content updates

GrahamBaird1 committed:
Content updates from legal

murrlipp committed:
Basic save and return added to adult journey

Still to include:
- Content design
- Timeout behaviour (TBC)
- Behaviour for multiple incorrect credential entries

murrlipp committed:
Content updates

murrlipp committed:
Merge branch 'master' into save-and-return-mvp

murrlipp committed:
Merge pull request #275 from CriminalInjuriesCompensationAuthority/save-and-return-mvp

Basic save and return added to adult journey

murrlipp committed:
Minor updates

murrlipp committed:
Save and return prototype updated

murrlipp committed:
Save and return prototype updates

murrlipp committed:
Tidying up and content amends

- Removed save and return from journey
- Updated link text in adult journey to include "(opens in new tab)"

murrlipp committed:
Content update

murrlipp committed:
Moved minor journey into main prototype

murrlipp committed:
Updated start page to match live version

murrlipp committed:
"Message bus down" concept screen added

GrahamBaird1 committed:
Message bus down content

Content for the confirmation page a user will see if the message bus is down and they do not automatically see/receive their application reference number from us.

murrlipp committed:
Content updates to minor and adult journeys

To match live service and UAT

GrahamBaird1 committed:
Declaration page content updates

Updated content with stakeholder feedback

GrahamBaird1 committed:
Updates to Declaration content

Small content tweaks to original Declaration screen commits.

GrahamBaird1 committed:
Adding anchor points to Declaration pages

For accessibility reasons. Screen readers need the link to show users using screen readers what they're agreeing to.

murrlipp committed:
Content updates to minor journey

murrlipp committed:
Routing and rep info screens added

Contextualisation to be completed
Check your answers screens to be updated

murrlipp committed:
Minor updates

murrlipp committed:
Routing fix

murrlipp committed:
Amending 'check your answers' screens

murrlipp committed:
Updating start screen

murrlipp committed:
Content updates

First pass at question content for reps journey

murrlipp committed:
Fixed routing error

murrlipp committed:
Content and routing updates

GrahamBaird1 committed:
3rd party reps prototype update

Changes to content to contextualise questions for user testing sessions on Tuesday 7 December.

murrlipp committed:
Fixed routing around crime reported date

murrlipp committed:
Merge branch 'master' into reps-journey-proto

murrlipp committed:
Merge pull request #276 from CriminalInjuriesCompensationAuthority/reps-journey-proto

Reps journey proto

murrlipp committed:
Removed redirect from index

murrlipp committed:
Adding contextualisation nunjucks

murrlipp committed:
Added 'download a PDF summary' to minors confirmation screen

murrlipp committed:
Fixing variable names

murrlipp committed:
Merge pull request #277 from CriminalInjuriesCompensationAuthority/reps-journey-proto

Reps journey proto

murrlipp committed:
Amending content, routing fixes

murrlipp committed:
Merge pull request #278 from CriminalInjuriesCompensationAuthority/reps-journey-proto

Amending content, routing fixes

murrlipp committed:
Capable adult section added

murrlipp committed:
Removed typo

murrlipp committed:
Typo fixed

murrlipp committed:
Updated error message

murrlipp committed:
Check your answers screen content updates

murrlipp committed:
Content updates to adult and minor journeys

murrlipp committed:
Declaration screen content updates

murrlipp committed:
Flow changes to account for all user groups

murrlipp committed:
Journey updates

murrlipp committed:
Content updates

First pass in Graham's absence

murrlipp committed:
Merge pull request #279 from CriminalInjuriesCompensationAuthority/reps-journey-proto

Merging reps journey prototype into main prototype

murrlipp committed:
Routing fixes, addition of overview screen

GrahamBaird1 committed:
3rd party reps content updates

Content updates across both adult and minor 3rd party reps journey.

GrahamBaird1 committed:
Add GitHub commit link

Added commit link to change log in prototype.

GrahamBaird1 committed:
Further content updates

Content updates to the adult and minor 3rd party reps journey.

murrlipp committed:
Added question

Added special equipment question to adults via reps journey

murrlipp committed:
Changed one question

GrahamBaird1 committed:
Adding DMI hint text to question

Hint text had disappeared from the screen. Adding this back to the question for context.

murrlipp committed:
Routing fix

murrlipp committed:
Added screens, disabled browser autofill

- Added non-CICA owned (GDS owned) screens
- Disabled browser autocomplete

murrlipp committed:
Bug fix

Multiple radio buttons could be selected

murrlipp committed:
Fixed declaration routing

murrlipp committed:
Routing and content changes

murrlipp committed:
Merge branch 'master' into reps-journey-proto

murrlipp committed:
Merge pull request #280 from CriminalInjuriesCompensationAuthority/reps-journey-proto

Reps journey proto

murrlipp committed:
Update index.html

murrlipp committed:
Added third party reps concept

murrlipp committed:
Fixed reps journey routing

murrlipp committed:
Removed reps journey iteration 1 from main flow

murrlipp committed:
Update index.html

murrlipp committed:
Updated reps journey concept routing

murrlipp committed:
Content updates to reps journey concept

murrlipp committed:
Content updates

murrlipp committed:
Adding files for cloud platform

murrlipp committed:
CP updates

murrlipp committed:
Amend commented out CP section

murrlipp committed:
Test push for cloud platform

murrlipp committed:
Update cd.yaml

murrlipp committed:
Test push

murrlipp committed:
moving .dockerignore

murrlipp committed:
Test push

murrlipp committed:
Test push

poornima-krishnasamy committed:
Add execute permissions

poornima-krishnasamy committed:
Merge pull request #281 from ministryofjustice/patch-01

Add execute permissions for start.sh

murrlipp committed:
Test push

murrlipp committed:
Merge branch 'master' of https://github.com/ministryofjustice/apply-for-compensation-prototype

murrlipp committed:
Iterating reps journey flow - content to be added

murrlipp committed:
Content updates to reps journey concept

murrlipp committed:
Add redirect

murrlipp committed:
Content updates

murrlipp committed:
Routing fixes

murrlipp committed:
Merge pull request #282 from ministryofjustice/reps-journey-proto-v2

Reps journey proto v2

murrlipp committed:
Separated contextualised screens in reps journey

To aid handover to dev

murrlipp committed:
Error fix

murrlipp committed:
Routing fixes - reps journey

murrlipp committed:
Update reps prototype flow and content

murrlipp committed:
Routing fix - rep journey

murrlipp committed:
Routing fix - rep journey

murrlipp committed:
Routing fix - rep journey

murrlipp committed:
Reps journey content updates

murrlipp committed:
Adding concept

murrlipp committed:
Content updates - reps journey concept

murrlipp committed:
Added end-to-end reps concept

Additional sections added to reps journey concept to make end-to-end journey for review.

murrlipp committed:
Added 'divert duplicate applications' concept

murrlipp committed:
Formatting fix

murrlipp committed:
Various updates

- Delayed applications
- Third party reps 'download a summary' concepts
- Duplicate applications concept updates

murrlipp committed:
Changed 'child' to 'victim' throughout

murrlipp committed:
Amended 'organisation name' reps flow

murrlipp committed:
Content update

murrlipp committed:
Routing fix

murrlipp committed:
Delete cd.yaml

murrlipp committed:
Create cd-cica-testing.yaml

To allow separate continuous deployment to prototype testing namespace on cloud platform.

murrlipp committed:
Rename kubernetes-deploy.tpl to kubernetes-deploy-cica-testing.tpl

murrlipp committed:
Update kubernetes-deploy.tpl

murrlipp committed:
Merge branch 'cica-testing' of https://github.com/ministryofjustice/apply-for-compensation-prototype into cica-testing

murrlipp committed:
Update kubernetes-deploy-cica-testing.tpl

murrlipp committed:
Update CP continuous deployment files

murrlipp committed:
Update kubernetes-deploy-cica-testing.tpl

murrlipp committed:
'Download a summary' concept added

murrlipp committed:
Merge pull request #283 from ministryofjustice/cica-testing

Cica testing

murrlipp committed:
Bug fixes

murrlipp committed:
Publishing concept screens for testing

murrlipp committed:
Test push

murrlipp committed:
Fixing cont. deployment files

murrlipp committed:
Update .gitignore

murrlipp committed:
Update index.html

murrlipp committed:
Content update

murrlipp committed:
Content updates

murrlipp committed:
Content updates

neil-stephen-mcgonigle committed:
added new application summary html files for download links

neil-stephen-mcgonigle committed:
added new application summary html files for download links

GrahamBaird1 committed:
Content changes to CICA testing branch

Changing content in CICA testing branch ahead of user testing 09/06/22.

GrahamBaird1 committed:
Merge branch 'cica-testing' of https://github.com/ministryofjustice/apply-for-compensation-prototype into cica-testing

GrahamBaird1 committed:
Merge branch 'cica-testing' of https://github.com/ministryofjustice/apply-for-compensation-prototype into cica-testing

GrahamBaird1 committed:
Additional content changes to testing branch

More content changes to the testing prototype ahead of user testing sessions on 09/06/22.

GrahamBaird1 committed:
Additional content changes to testing branch

More content changes to the testing prototype ahead of user testing sessions on 09/06/22.

GrahamBaird1 committed:
Final content tweaks to prototype

Few last-minute content updates to the prototype ahead of testing sessions on 09/06/22.

GrahamBaird1 committed:
Final content tweaks to prototype

Few last-minute content updates to the prototype ahead of testing sessions on 09/06/22.

DanGuyMOJ committed:
add unique values to the rep's address page

DanGuyMOJ committed:
add unique values to the rep's address page

DanGuyMOJ committed:
Remove money section from check your answers pages (reps journey)

DanGuyMOJ committed:
Remove money section from check your answers pages (reps journey)

GrahamBaird1 committed:
Content updates to ensure prototype is current

Content updates made across the download and incident delay screens.

GrahamBaird1 committed:
Content updates to ensure prototype is current

Content updates made across the download and incident delay screens.

DanGuyMOJ committed:
Remove money section from concepts/third-party-reps/download-a-copy/print-screen-on-cya/check-your-answers

DanGuyMOJ committed:
Edits to divert-duplicate-apps including restructure of pages and routes.js

DanGuyMOJ committed:
Add CYA and declaration to divert-duplicate-apps prototype

DanGuyMOJ committed:
Update transition.html

DanGuyMOJ committed:
Create 2022-07.html

DanGuyMOJ committed:
build contact CICA journey and add month filter

DanGuyMOJ committed:
add contact-cica pt to concepts tab of index and add value attributes to all form fields

DanGuyMOJ committed:
update duplicate-apps content with Graham: declaration; confirmation; cya and add overview page as start page

DanGuyMOJ committed:
add missing gov pages to start of duplicate-apps, add additional form pages and add update logic on declaration page

DanGuyMOJ committed:
fix bug on duplicate-apps and finesse content before testing

DanGuyMOJ committed:
Update _2_victim_details.html

DanGuyMOJ committed:
update extra instances of 'Update an existing claim'

DanGuyMOJ committed:
Save and return V2 prototype

DanGuyMOJ committed:
Update mobile-email.html

DanGuyMOJ committed:
S&R amends based on Efi's comments

DanGuyMOJ committed:
S&R V2 - replace task-list with context page

DanGuyMOJ committed:
add rep detail pages to S&R v2

DanGuyMOJ committed:
correct content on extra rep-details pages included in S&R v2

GrahamBaird1 committed:
Update save and return content (return only)

Updating content for save and return journey – I've only updated the 'return' journey content.

Dan will be updating the 'save' journey content.

DanGuyMOJ committed:
Add content changes to save part of s&r v2

DanGuyMOJ committed:
Merge branch 'master' of https://github.com/ministryofjustice/apply-for-compensation-prototype

DanGuyMOJ committed:
Finesse content changes to s&r v2 before first round of testing

DanGuyMOJ committed:
S&R add timeout pages and fix content errors

DanGuyMOJ committed:
Update timeout.html

DanGuyMOJ committed:
fix govuk guidance buttons (prev/next) on S&R v2

DanGuyMOJ committed:
make leave application btns more relevant on S&Rv2

DanGuyMOJ committed:
update to s&r dashboard

DanGuyMOJ committed:
replace confirmation panel with notification panel once user has set up S&R

DanGuyMOJ committed:
add session timeout journey

DanGuyMOJ committed:
add current GDS sign in journey

jasonBirchall committed:
fix(kubernetes): change template to reflect ingress changes

DanGuyMOJ committed:
Update application.js

jakemulley committed:
Update deployment workflow to replace use of long-lived credentials

DanGuyMOJ committed:
Merge pull request #288 from ministryofjustice/update-workflow-2

Update deployment workflow to replace use of long-lived credentials
